syntax = "proto3";
package Jubo.JuLiao.IM.Wx.Proto; //命名空间约定
import "TransportMessage.proto";

message TalkToFriendTaskMessage {
    string WeChatId = 1; // 所属微信号
    string FriendId = 2; // 发送给那个好友
    EnumContentType ContentType = 3; // 发送消息类型 Text Picture Voice Video Link File NameCard WeApp Quote Emoji ShiPinHao
    bytes Content = 4; // 发送消息内容 文本; 图片，视频，语音，文件url; 链接json; 名片wxid; Emoji的md5或Emoji的详细json
    string Remark = 8; // 其他备注信息，群聊@别人；Quote（引用消息）：引用消息的msgSvrId字符串
    int64 MsgId = 9; // 发送给手机端的时候需要赋值，用于TalkToFriendTaskResultNotice中
    bool Immediate = 10; //立即发送（用于群发消息时优先发送聊天消息）
}